CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # TimescaleDB Changelog
 
+## 0.3.0 (2017-07-31)
+
+**High-level changes**
+* "Upserts" are now supported via normal `ON CONFLICT DO UPDATE` syntax. However, `ON CONFLICT ON CONSTRAINT` is not yet supported.
+* Improved support for user-defined triggers on hypertables
```

Comment:
Improved support for user-defined triggers on hypertables.  Now handles both `INSERT BEFORE` and `INSERT AFTER` triggers, and triggers can be named arbitrarily (before, a \_0_ prefix was required to ensure correct execution priority).